plugins/renepay/mods.c

```diff
@@ -253,6 +253,18 @@ static struct command_result *previous_sendpays_done(struct command *cmd,
 		// max_partid for the pending_group_id
 		payment->next_partid = max_pending_partid + 1;
 
+		for (size_t j = 0; j < tal_count(pending_routes); j++) {
+			route_pending_register(payment->routetracker,
+					       pending_routes[j]);
+		}
+		if (!routetracker_get_amount(payment->routetracker,
+					     &payment->total_delivering,
+					     &payment->total_sent))
+			plugin_err(pay_plugin->plugin,
+				   "(%s:%d) routetracker_get_amount failed "
+				   "probably due to an amount_msat overflow.",
+				   __PRETTY_FUNCTION__, __LINE__);
```

Comment:
I did not make my reseach but we should make sure that this macros are present in all kind of compilers

Reply:
Never thought of this before. I have them all over the code.

Reply:
>Never thought of this before. I have them all over the code.

well, I think we are safe, till now there are no compiled bug reports, so I think we are safe. I like the line number idea in the log a, so all good

Reply:
Well, `__func__` and `__LINE__` are standards (https://gcc.gnu.org/onlinedocs/cpp/Standard-Predefined-Macros.html).
The `__PRETTY_FUNCTION__` macro seems to be introduced by gcc, but it is also supported by clang; this one is a substitute for `__func__` for long function names in C++ (my bad habits from C++ creeping into clightning).
I'll make a following PR to substitute all `__PRETTY_FUNCTION__` with `__func__`.